lib/lightspeed_restaurant/inventory_product.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'lightspeed_restaurant/base'
+require 'lightspeed_restaurant/operations/list'
```

Comment:
needed?

Reply:
should have been find, but I guess it was already required somewhere else. Updated it to `find`